VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
             @Override
             public void onClick(View v) {
                 if (comment.getChildren().isEmpty()) {
-                    router.showCourseDiscussionAddComment(context, comment);
+                    if (!discussionThread.isClosed())
+                        router.showCourseDiscussionAddComment(context, comment);
```

Comment:
Note that adding an empty click handler to a view might add unwanted sound/accessibility effects.
> On 06-Oct-2015, at 6:57 PM, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java <https://github.com/edx/edx-app-android/pull/382#discussion_r41266518>:
> 
> > @@ -219,10 +219,11 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
> >              @Override
> >              public void onClick(View v) {
> >                  if (comment.getChildren().isEmpty()) {
> > -                    router.showCourseDiscussionAddComment(context, comment);
> > +                    if (!discussionThread.isClosed())
> > +                        router.showCourseDiscussionAddComment(context, comment);
> yes, but it here it requires a lesser amount of code lines to do the thing.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/382/files#r41266518>.
>